apps/portlet-documents/src/main/webapp/js/transferRulesService.js

```diff
@@ -0,0 +1,18 @@
+export function getTransfertRulesDownloadDocumentStatus() {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/transferRules/getTransfertRulesDocumentStatus`, {
+    method: 'GET',
+    credentials: 'include',
+  }).then((resp) => {
+    if (resp && resp.ok) {
+      return resp.json();
+    }
+    else {
+      throw new Error ('Error when getting transfer rules download documents status');
+    }
+  }).then(data => {
+    if (data) {
+      const downloadStatusPromise =  data.downloadDocumentStatus !== 'true';
+      return Promise.resolve(downloadStatusPromise);
+    }
```

Comment:
It should be `downloadDocumentStatus === 'true'` to be able to download, isn't it ?
```suggestion
    return data && data.downloadDocumentStatus === 'true';
```

Reply:
> It should be `downloadDocumentStatus === 'true'` to be able to download, isn't it ?
When `downloadDocumentStatus === 'true' the download action is disabled, we suspend the download.
> **BTW**: no need to create a `new Promise`, since the value will be returned with inside a `Promise` already.
Done!